superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -955,6 +955,18 @@ export const controls = {
     description: 'Whether to include a time filter',
   },
 
+  filter_order: {
+    type: 'SelectControl',
+    multi: true,
+    label: 'Order by optional value',
+    default: [],
+    mapStateToProps: state => ({
+      choices: (state.datasource) ? state.datasource.order_by_choices : [],
```

Comment:
I think, in most cases, users know which option they want to choose, and we just need to help them to find it quickly. so, be `alphabetically` and be `reverse order` are used more often than `by metric descending`. 
For example, area(alphabetically) and period(like year-month in reverse order) are used regularly in sales analysis.